src/asim/configs/resident/vehicle_type_choice.yaml

```diff
@@ -24,9 +24,11 @@ SIMULATION_TYPE: interaction_simulate
 # VEHICLE_TYPE_DATA_FILE: vehicle_type_data.csv
 VEHICLE_TYPE_DATA_FILE: vehicle_type_data_extended.csv
 
+REQUIRE_DATA_FOR_ALL_ALTS: False
+
 # age is computed as (1 + FLEET_YEAR - vehicle_year)
 # FLEET_YEAR: 2017
-FLEET_YEAR: 2017
+FLEET_YEAR: 2023
```

Comment:
@dhensle please note this will be updated to the model scenario year based on Ali's work here - https://github.com/SANDAG/ABM/blob/ABM3_develop/src/asim/scripts/scenarioManagement/scenManagement.py#L121. Could you please confirm that this should be 2023 and not 2022?

Reply:
Yes, I got the base year incorrect.  I have changed it from 2023 to 2022.